index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Md. Porag Sarkar: Portfolio</title>
+</head>
+
+<body>
+    <div class="container-parent">
+
+        <header class="top-navigation">
+            <div class="container-logo">
+                <p class="logo">My Logo</p>
+            </div>
+            <img class="menu-icon" src="./assets/mobile/Icon - Menu.svg" alt="Menu Button">
+
+        </header>
+
+        <main class="main">
+            <section class="headline">
+                <div class="text-primary">
+                    <h2 class="primary-one">I’m Porag. Glad to see you!</h2>
+                    <p class="primary-two">I’m a software developer! I can help you build a product , feature or website
+                        Look through some of my work and experience! If you like what you see and have a project you
+                        need coded, don’t hesitate to contact me.</p>
+                </div>
+
+                <div class="social-media">
+                    <p class="text-media-supporting">LET’S CONNECT</p>
```

Comment:
- [ ] Please use a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)` for the media icons title. This way will be semantically correct and the screen readers and other technologies will thank you.